python/pyarrow/feather.py

```diff
@@ -175,15 +123,38 @@ def write_feather(df, dest):
         Dataframe to write out as feather format.
```

Comment:
update there that it also accepts a table? 

(actually, in the parquet module, it uses a different naming scheme, i.e. `write_table`)